data/src/main/java/com/example/rickandmorty/data/koin/SharedPrefsModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.rickandmorty.data.koin
+
+import com.example.rickandmorty.data.service.PreferenceServiceImpl
+import com.example.rickandmorty.domain.service.LanguageService
+import com.example.rickandmorty.domain.service.NightModeService
+import org.koin.core.module.dsl.bind
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.module
+
+val sharedPrefsModule = module {
+    singleOf(::PreferenceServiceImpl)
```

Comment:
вот тут у тебя преференсы отдаются по Impl, так же тут если у тебя в одном сервисе несколько фичей (например те же локаль и темная тема), то можно сделать разные интерфейсы для этого
```suggestion
    singleOf(::PreferenceServiceImpl) { bind<PreferenceService>() }
```